src/client/utils/url.js

```diff
@@ -1,39 +1,22 @@
-import sharedUrlUtil from '../../utils/url';
-import { isCrossDomainWindows } from '../utils/dom';
-import { getAttribute as nativeGetAttribute } from '../sandbox/native-methods';
-import { DOM_SANDBOX_STORED_ATTR_POSTFIX, DOCUMENT_CHARSET } from '../../const';
+import sharedUrlUtils from '../../utils/url';
```

Comment:
Not exactly, urlUtils contain a lot of client-specific code. I sure we will solve it in the future.